op-node/rollup/derive/attributes.go

```diff
@@ -8,9 +8,9 @@ import (
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/core/types"
 
+	"github.com/ethereum-optimism/optimism/op-bindings/predeploys"
```

Comment:
How about adding alias for oppredeploys import? (in all places) It's quite confusing if seeing `predeploys.xxx`